src/Microsoft.ServiceFabric.Services/Runtime/StatefulServiceReplicaAdapter.cs

```diff
@@ -195,11 +195,31 @@ async Task IStatefulServiceReplica.CloseAsync(CancellationToken cancellationToke
 
             await this.CloseCommunicationListenersAsync(cancellationToken);
             await this.CancelRunAsync();
+            ServiceTrace.Source.WriteInfoWithId(

```

Comment:
Would it be useful logging replica/partition id here too?

Reply:
its already logged in the type field.

Reply:
ok